collection-scripts/gather_ceph_resources

```diff
@@ -78,6 +78,7 @@ ceph_commands+=("ceph tell mds.ocs-storagecluster-cephfilesystem:0 dump_historic
 ceph_commands+=("ceph tell mds.ocs-storagecluster-cephfilesystem:0 session ls")
 ceph_commands+=("ceph tell mds.ocs-storagecluster-cephfilesystem:0 client ls")
 ceph_commands+=("ceph tell mds.ocs-storagecluster-cephfilesystem:0 damage ls")
+ceph_commands+=("ceph auth dump-keys")
```

Comment:
This command **only** works with `--format=json` or `--format=json-pretty` (preferred)

```suggestion
ceph_commands+=("ceph auth dump-keys --format=json-pretty")
```

Reply:
no worries, that won't hamper the collection, it just sets a log msg if any resource is missing or there is an error